public/controllers/agent/agents.js

```diff
@@ -193,6 +196,27 @@ export class AgentsController {
     this.$scope.updateSelectedItem = i => (this.$scope.selectedItem = i);
     this.$scope.getIntegration = list =>
       this.configurationHandler.getIntegration(list, this.$scope);
+
+
+    this.$scope.downloadCsv = async data_path => {
```

Comment:
This method is duplicated and placed in the wrong place. Take a look at https://github.com/wazuh/wazuh-kibana-app/blob/6045fb4d27728780892e474455b85b72e0843631/public/controllers/agent/agents.js#L543, also use _camelCase_ please